src/crud.rs

```diff
@@ -98,7 +140,7 @@ macro_rules! crud {
           #[doc="This just calls [inner_create_" $i "] with `send_signal` as `true`."]
           #[hdk_extern]
           pub fn [<create_ $i>](entry: $crud_type) -> ExternResult<$crate::wire_element::WireElement<[<$crud_type>]>> {
-            [<inner_create_ $i>](entry, true)
+            [<inner_create_ $i>](entry, true, None)
```

Comment:
I wonder what we should set as a default... we could in fact use the entry type here to provide that default behaviour